docs/scripts/convert_code_to_documentation.py

```diff
@@ -64,10 +54,9 @@
 build_dir = pathlib.Path("docs/build")
 sdk_dir = pathlib.Path("docs/sdk")
 autosummary_dir = pathlib.Path("docs/_autosummary")
-destination_dir = pathlib.Path(DESTINATION_DIR)
 
 # Collect all of the directories and delete them if they still exist.
-directories = [sdk_dir, autosummary_dir, build_dir, destination_dir]
+directories = [sdk_dir, autosummary_dir, build_dir]
```

Comment:
Because they are not necessary 😄 We can also just keep them, then the documentation folder will probably be slightly larger, but I do not think that there is a huge difference. Would however make the code a bit more leaner. What do you prefer?

Reply:
I'm confused a bit I have to say :D

So my attempt to logical thinking here is: Sphinx creates the content during a run; we don't need it, but apparently sphinx needs it (otherwise it wouldn't have created the stuff in the first place); if we delete it, it must create the stuff again in the next run (same argument as for the other folder). So ... what is wrong in this line of argument?

Reply:
The reason for that was that we are interested in only the documentation itself - not all of the stuff that sphinx created. Hence, when moving the documentation to another place, there was not sense to also move all of the stuff that sphinx created: We only cared for the actual HTML/Markdown files of the documentation. Hence, the other stuff was deleted.
That is at least how I remember the line of thinking back then. But I get from this discussion that you'd prefer to just not delete stuff?

Reply:
Ok, so then my conclusion would be that the other folders should also stay, because it can only help to speedup things. @Scienfitz, opinions?

Reply:
Will be removed.